### PR TITLE
add loading bar for cancer gene list

### PR DESCRIPTION
**Pull request by JiaoJiao123**

### CODE REVIEW

File: app/styles/_cancerGenes.scss [hunk 1]
```diff
@@ -3,3 +3,7 @@
 .cancerGenes {
   font-size: 14px;
 }
+.loadingStyle {
```
Comment (zhx828) [inline app/styles/_cancerGenes.scss]: The class name should be named a-b. Not camel case.

**Comment by JiaoJiao123** [comment]
@zhx828 PR updated about making loading styles uniformed

File: app/scripts/controllers/cancerGenes.js [hunk 2]
```diff
@@ -37,45 +37,44 @@ angular.module('oncokbStaticApp')
...
+                default:
+                    break;
+                }
+            });
+            $scope.cancerGeneList = tempData;
+            $scope.doneLoading = true;
+        }, function(error) {
+            $scope.cancerGeneList = [];
+            $scope.doneLoading = true;
+            console.log('Error happened', error);
```
Comment (zhx828) [inline app/scripts/controllers/cancerGenes.js]: Remove the console log

Comment (JiaoJiao123) [inline app/scripts/controllers/cancerGenes.js]: sure

**Comment by JiaoJiao123** [comment]
@zhx828 updated

**Pull request merged**